doc/nrf/app_dev/device_guides/coprocessors/power_management.rst

```diff
@@ -0,0 +1,208 @@
+.. _hpf_power_management:
+
+Power management
+################
+
+.. caution::
+
+   The High-Performance Framework (HPF) support in the |NCS| is :ref:`experimental <software_maturity>` and is limited to the nRF54L15 device.
+
+The following page outlines the approaches and mechanisms employed to ensure efficient power management in High-Performance Framework (HPF) systems, focusing on resource allocation, system states, and interactions between different components.
+
+Power management strategies
+***************************
+
+Power Management in HPF should be considered from two perspectves: the Host controller and the HPF firmware.
+
+Host controller
+===============
+
+The Host is responsible for managing power state of the following resources:
+
+* FLPR core with HPF firmware running on it
+* RAM
+
+To achieve optimal power state, you must use the `Device Runtime Power Management`_ and `System Power Off`_.
```

Comment:
It should be shown that the power management is an architecture design and SW components are not there yet.
@mif1-nordic is working on adding that to this page.